atlp-rwanda/atlp-pulse-fn
# feat:implememtation  attendance

# PR Description
Attendance
a coordinator should see the list of attendance
 
- **attendance distribution**
     - 0: Didn't attend and didn't communicate
     - 1: Didn't attend but communicated
     - 2: attended
 
Attendance statistics
 
# How has this been tested?
- For browser
  - Go [Here](https://metron-devpulse-git-ft-attendance-metron.vercel.app) and login as Coordinator
     - email: alineuwera00@gmail.com
     - password: Test@12345
     
  - click on attendance you will be able to see attendance based on week and corresponding statistics
  - To record attendance follow this:
     - after click on attendance click on submit attendance.
     - choose week, day and assign value to trainnee then save
# Number of Commits

1

# Screenshots (If appropriate)
<img width="797" alt="attendance" src="https://github.com/atlp-rwanda/atlp-pulse-fn/assets/112135715/19f6f2e6-cbac-4f35-b7b8-59a4e6f5671e">

# Please check this Checklist before you submit your PR:
